coordinator/src/services/translation/request/agent-unattended.ts

```diff
@@ -25,6 +25,33 @@ export function createAuthor(
   return author
 }
 
+export function createAuthorForWithdraw(
+  practitionerRole: fhir.PractitionerRole,
+): hl7V3.AuthorPersonSds {
+
+  const sdsRoleProfileId = getIdentifierValueForSystem(
+    practitionerRole.identifier,
+    "https://fhir.nhs.uk/Id/sds-role-profile-id",
+    'Task.contained("PractitionerRole").identifier("value")'
+  )
+
+  if (isReference(practitionerRole.practitioner)) {
+    throw new errors.InvalidValueError(
+      "practitionerRole.practitioner should be an Identifier",
```

Comment:
minor but I think this should be:

`"practitionerRole.practitioner should be an Value Identifier"`

because identifiers can either be `value identifiers` or `reference identifiers`

if there's nothing else don't worry about making this change

Reply:
think thats just naming in our type definitions jack. the fhir message would be:
```
{
  "resourceType": "PractitionerRole",
  "practitioner": {
    "identifier": []
  }
}
```